### DISCUSSION TITLE
github linux loginds - do I have to use Windows so your website works?

### DISCUSSION THREAD

**Discussion by vakills** [community/community#89902] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I would have thought that the geeks here would figure out how people can login without reducing/killing all their personal security.  Are you the federal government?  That is their attitude also.  Shame on you!

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @vakills! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

With that being said, I do want to add that you are more likely to get a useful response if you are explicit about what your question entails, giving a few more details might help someone answer your question. 

I've gone ahead and moved this to the correct category for you. Good luck!
